gtsam_unstable/discrete/tests/testSudoku.cpp

```diff
@@ -185,29 +164,24 @@ TEST_UNSAFE( Sudoku, extreme)
   index.outputMetisFormat(os);
 #endif
 
-  //sudoku.printSolution(); // don't do it
+  // sudoku.printSolution(); // don't do it
 }
 
 /* ************************************************************************* */
-TEST_UNSAFE( Sudoku, AJC_3star_Feb8_2012)
-{
-  Sudoku sudoku(9,
-      9,5,0, 0,0,6, 0,0,0,
-      0,8,4, 0,7,0, 0,0,0,
-      6,2,0, 5,0,0, 4,0,0,
+TEST_UNSAFE(Sudoku, AJC_3star_Feb8_2012) {
+  Sudoku sudoku(9, 9, 5, 0, 0, 0, 6, 0, 0, 0, 0, 8, 4, 0, 7, 0, 0, 0, 0, 6, 2,
+                0, 5, 0, 0, 4, 0, 0,
 
-      0,0,0, 2,9,0, 6,0,0,
-      0,9,0, 0,0,0, 0,2,0,
-      0,0,2, 0,6,3, 0,0,0,
+                0, 0, 0, 2, 9, 0, 6, 0, 0, 0, 9, 0, 0, 0, 0, 0, 2, 0, 0, 0, 2,
+                0, 6, 3, 0, 0, 0,
 
-      0,0,9, 0,0,7, 0,6,8,
-      0,0,0, 0,3,0, 2,9,0,
-      0,0,0, 1,0,0, 0,3,7);
+                0, 0, 9, 0, 0, 7, 0, 6, 8, 0, 0, 0, 0, 3, 0, 2, 9, 0, 0, 0, 0,
```

Comment:
LGTM. Since these represent a 9x9 sudoku grid, maybe these few lines are more readable with the prior formatting and a nolint declaration?

Reply:
Good catch! Will do

Reply:
Done. Will you approve? PS - if you trust people to make the change, approve immediately. Reduces back and forth. But you have to be confident, of course.